src/assets/styles/_base.scss

```diff
@@ -34,10 +34,14 @@ h2,
 h3,
 h4,
 h5,
-h6 {
+h6,
+section h1 {
   line-height: 1.3;
   font-weight: var(--font-weight-semi-bold);
 }
+section h1 {
+  font-size: var(--type-xxxl, 32px);
+}
```

Comment:
**Issue**: This and the above shouldn’t be a base style. It’s specific to the component that uses these section elements and should be defined there.

Reply:
If we overwriting `h1` (as we are), then we should also overwrite `section h1` here:

![Screenshot 2023-07-17 at 17 15 51](https://github.com/kumahq/kuma-gui/assets/554604/aa6e9a73-adfa-40d3-a47e-67077ffe80b3)

They are both styles that are applied by the browser

Reply:
What user agent styles exist does not determine for what we need to add base styles. `section` is functionally equivalent with a `div` besides what’s specified in the HTML standards, I don’t see any reason at all that we ever want a base style for `section h1`.

Besides the use introduced here, we have none that need it. And the use here should be handled by component styles, not base styles.

Reply:
I've moved these into the component